src/include/common/copier_config/copier_config.h

```diff
@@ -19,18 +19,21 @@ struct CSVReaderConfig {
     char listBeginChar;
     char listEndChar;
     bool hasHeader;
+    bool parallel;
```

Comment:
Not really related to this PR. I wonder if we should merge CSVReaderConfig with ReaderConfig at some point. Each reader can access a subset of fields from ReaderConfig class.

Reply:
Some attributes can be shared. `Parallel` definitely should be. Others though... don't make sense, right?